pipeline.sh

```diff
@@ -214,7 +222,7 @@ fi
 if [ ! -z "${FORCE}" ]; then
 	PREPROCESS_ARGS+=" -f"
 fi
-sh "${SCRIPT_DIR}"/pipeline_preprocess.sh ${PREPROCESS_ARGS}
```

Comment:
I prefer SCRIPT_DIR to SCRIPT_SUBMIT_DIR, since your not submitting only makes sense in a job execution context.

Reply:
Suggested change was implemented.